styles.css

```diff
@@ -0,0 +1,1985 @@
+    color: #eaecf0;
+    min-width: auto;
+    width: 327px;
+    height: auto;
+    margin: 24px;
+    padding-bottom: 80px;
+    padding-left: 16px;
+    padding-right: 16px;
+    padding-top: 15px;
+    /*position: relative;*/
+    /*right: 10px;*/
+}
+
+label {
+    font-size: 22px;
+    letter-spacing: 1px;
+    display: block;
+}
+
+input {
+    height: 50px;
+    width: 291px;
+    margin-bottom: 15px;
+    margin-top: 12px;
+    margin-right: 0px;
+}
+
+input#age {
+    width: 75px;
+    display: block;
+    margin-top: 5px;
+}
+
+textarea#selfdescribe {
+    margin-bottom: 40px;
+    width: 291px;
+}
+
+textarea#powers {
+    height: 175px;
+    width: 291px;
+}
+
+
+legend {
+    font-family: 'BenchNine', sans-serif;
+    font-size: 22px;
+    font-weight: 700;
+}
+
+p {
+    font-size: 20px;
+    font-weight: 400;
+    color: #D1D5DE;
+    line-height: 22px;
+    margin-bottom: 5px;
+    margin-top: 5px;
+}
+
+.abilities {
+    border-bottom: 4px solid #163346;
+    color: #eaecf0;
+    font-family: 'Teko', sans-serif;
+    font-size: 40px;
+    font-weight: 400px;
+    height: 35px;
+    letter-spacing: 1px;
+    margin-bottom: 30px;
+    width: 235px;
+}
+
+.rate {
+    color: #eaecf0;
+    font-size: 22px;
+    margin-bottom: 10px;
+    margin-top: 10px;
+}
+
+.svg {
+    background-color: transparent;
+    display: inline-block;
+    left: 60px;
+    margin: 0;
+    margin-left: 75px;
+    padding: 0;
+    width: 70%;
+    /*right: 45px;*/
+    /*padding-bottom: 40px;*/
+}
+
+.strength {
+    height: 100px;
+    width: 111px;
+    /*align-content: center;*/
+    margin-bottom: 19px;
+    margin-left: 0px;
+    /*margin-top: 37px;*/
+    /*margin-right: 0px;*/
+    padding: 0px;
+    /*position: relative;*/
+    /*left: 85px;*/
+}
+
+.speed {
+    border: 0px;
+    height: 100px;
+    margin-left: 0;
+    margin-right: 0;
+    margin-top: 40px;
+    width: 99px;
+    padding: 0;
+    /*position: relative;*/
+    /*left: 90px;*/
+}
+
+.intelligence {
+    border: 0px;
+    height: 110px;
+    /*min-width: auto;*/
+    margin-left: 0;
+    margin-right: 0;
+    margin-top: 40px;
+    width: 110px;
+    /*position: relative;*/
+    padding: 0;
+    /*left: 84px;*/
+}
+
+select#strength {
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    appearance: none;
+    display: block;
+    font-size: 20px;
+    height: 73px;
+    /*left: 30px;*/
+    /*right: 44px;*/
+    width: 70%;
+    text-align: center;
+    position: relative;
+}
+
+
+select#speed {
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    appearance: none;
+    font-size: 20px;
+    height: 73px;
+    width: 70%;
+}
+
+select#intelligence {
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    appearance: none;
+    border: 5px;
+    /*display: block;*/
+    font-size: 20px;
+    /*left: 30px;*/
+    margin-bottom: 104px;
+    /*right: 44px;*/
+    height: 73px;
+    width: 70%;
+    /*position: relative;*/
+}
+
+
+input#confirm {
```

Comment:
This is a good example of a more unique element -- it shares many things in common with the other checkboxes, but it's got some of its own styling, too. Again, I'd leverage what it has in common and only write rules here for what is different .